### PR TITLE
Run PEP8 on content

### PR DESCRIPTION
**Pull request by sptramer**
Hello! The Azure documentation team (Content & Learning) is updating existing
Python samples to conform to PEP8. This consists of (only) running `autopep8`
on content. Even if this PR contains only whitespace changes, please accept it.

If you have questions about the rationale behind this change, please email
Microsoft alias `sttramer`.

### CODE REVIEW

File: samples/search/autosuggest_samples.py [hunk 1]
```diff
@@ -9,19 +9,23 @@
...
     try:
-        suggestions = client.auto_suggest(query="Satya Nadella")  # type: Suggestions
+        suggestions = client.auto_suggest(
+            query="Satya Nadella")  # type: Suggestions
 
         if suggestions.suggestion_groups:
             print("Searched for \"Satya Nadella\" and found suggestions:")
-            suggestion_group = suggestions.suggestion_groups[0]  # type: SuggestionsSuggestionGroup
+            # type: SuggestionsSuggestionGroup
+            suggestion_group = suggestions.suggestion_groups[0]
```
Comment (lmazuel) [inline samples/search/autosuggest_samples.py]: type annotations are required to be after the line, might be a autopep8 issue

Comment (sptramer) [reply samples/search/autosuggest_samples.py]: Seems like it. Also we're going to be trying to move all samples to be compatible with Python 3.6 so official type hinting support should be used.

**Comment by sptramer** [comment]
Looks like this is failing due to a CI issue, not autopep8.

File: samples/search/autosuggest_samples.py (hunk 1 above)
Comment (lmazuel) [reply samples/search/autosuggest_samples.py]: @sptramer if you do so, that would mean 2.7 or even 3.5 people won't be able to execute the sample, is that really what we want???

Comment (sptramer) [reply samples/search/autosuggest_samples.py]: Yes. Python 2 is being sunsetted permanently on Jan 1 2020, and research indicates that the vast majority of Python users are on 3.6.

Comment (lmazuel) [reply samples/search/autosuggest_samples.py]: Research maybe but not metrics :). Please discuss this point with my PM, I'll start an email thread

Comment (lmazuel) [reply samples/search/autosuggest_samples.py]: So, I don't want to let your PR open too long, I don't you to have conflict to solve. I started an email thread to discuss 2.7 support, in the meantime I revert this particular so I can merged it. If you feel really strong about this particular, line, feel free to open a new PR with just this line.
Thanks

**Pull request merged**